PlantPlaces23FS7024001/Pages/Index.cshtml.cs

```diff
@@ -40,16 +40,16 @@ public void OnGet()
             return await Task.Run(async () =>
             {
 
-               var config = new ConfigurationBuilder()
-                .AddUserSecrets<Program>()
-                .Build();
+                var config = new ConfigurationBuilder()
+                 .AddUserSecrets<Program>()
+                 .Build();
                 string apikey = config["weatherapikey"];
 
                 Task<HttpResponseMessage> plantTask = httpClient.GetAsync("https://plantplaces.com/perl/mobile/viewplantsjsonarray.pl?WetTolerant=on");
-                Task<HttpResponseMessage> task = httpClient.GetAsync("https://plantplaces.com/perl/mobile/specimenlocations.pl?Lat=39.1455&Lng=-84.509&Range=0.5&Source=location");
+                Task<HttpResponseMessage> specimenTask = httpClient.GetAsync("https://plantplaces.com/perl/mobile/specimenlocations.pl?Lat=39.1455&Lng=-84.509&Range=0.5&Source=location");
                 Task<HttpResponseMessage> weatherTask = httpClient.GetAsync("https://api.weatherbit.io/v2.0/current?&city=Cincinnati&country=USA&key=" + apikey);
-                
-                HttpResponseMessage result = task.Result;
+
+                HttpResponseMessage result = specimenTask.Result;
```

Comment:
Variable should have descriptive names, as described here:
https://dev.to/tanisha03/clean-code-part-1-meaningful-names-4khp